app/components/Settings/Menu/Menu.js

```diff
@@ -17,6 +17,10 @@ const Menu = ({ setActiveSubMenu }) => (
       <FormattedMessage {...messages.locale} />
       <FaAngleRight />
     </li>
+    <li className={styles.fiat} onClick={() => setActiveSubMenu('theme')}>
+      <span>Theme</span>
```

Comment:
This string needs to be rendered using the `<FormattedMessage>` component in order to ensure it is translatable.

See https://github.com/yahoo/react-intl/wiki/Components#formattedmessage